.gitignore

```diff
@@ -11,3 +11,5 @@ coverage.out
 tags
 temp_parser_file
 y.output
+vendor
+_vendor
```

Comment:
Hi @coocood ,

oh.., I overlooked this change. 
But this folders always annoy me. How you avoid these folders to be commited.

Regards,
Yu